specifications/api/swagger.yaml

```diff
@@ -2940,3 +2996,29 @@ paths:
       parameters:
         - *parametersAddressId
       responses: *responsesInspectAddress
+
+  /settings:
+    put:
+      operationId: putSettings
+      tags: ["Settings"]
```

Comment:
The tag needs to be added also in one of the tag groups (https://github.com/input-output-hk/cardano-wallet/blob/master/specifications/api/swagger.yaml#L2177) otherwise it will be not visible in https://input-output-hk.github.io/cardano-wallet/api/edge/.

I think `Miscellaneous` would be good place...